OneCommunityGlobal/HighestGoodNetworkApp
# Ek task edit suggest

### Merging broke parts of this implementation. Working on a fix.

Implements the functionality to suggest edits to existing tasks. Only managers/mentors can suggest edits. Admins/owners can edit tasks without needing to suggest anything. Volunteers can only view tasks.

To test:

1. Navigate to a task, and click View/Suggest/Edit depending on your role.
2. Make the changes you want to suggest, and click the Update button at the bottom of the modal.
3. Logout, and login as an admin/owner.
4. Click the red circle with the number on it in the navbar (this should display a count of the number of pending task edit suggestions. Only admins/owners should see this button.
5. Click on one of the rows to open a modal displaying the task edit suggestions. There are three buttons at the bottom.

- Check - approves the task edit suggestion. You should be able to view the approved changes on the dashboard or task page.
- Pen - edit the task edit suggestion before approval. You should be able to make any edits to the suggestion and then approve them.
- Trash - rejects the task edit suggestion. The task should remain unchanged.
